go.mod

```diff
@@ -5,8 +5,12 @@ go 1.14
 require (
 	github.com/aws-controllers-k8s/runtime v0.6.0
 	github.com/aws/aws-sdk-go v1.38.11
+	github.com/ghodss/yaml v1.0.0
```

Comment:
need to add attributions, please create a task

Reply:
Fortunately, all these packages are already in attributions

Just need to add `go.uber.org/zap` as package dependency list - https://github.com/aws-controllers-k8s/sagemaker-controller/blob/main/ATTRIBUTION.md#package-dependencies